test_communication/test/message_fixtures.hpp

```diff
@@ -119,4 +120,68 @@ get_messages_static_array_primitives()
   return messages;
 }
 
+std::vector<test_communication::DynamicArrayPrimitives::Ptr>
+get_messages_dynamic_array_primitives()
+{
+  std::vector<test_communication::DynamicArrayPrimitives::Ptr> messages;
+  {
+    auto msg = std::make_shared<test_communication::DynamicArrayPrimitives>();
+    msg->bool_values = {};
+    msg->byte_values = {};
+    msg->char_values = {};
+    msg->float32_values = {};
+    msg->float64_values = {};
+    msg->int8_values = {};
+    msg->uint8_values = {};
+    msg->int16_values = {};
+    msg->uint16_values = {};
+    msg->int32_values = {};
+    msg->uint32_values = {};
+    msg->int64_values = {};
+    msg->uint64_values = {};
+    msg->string_values = {};
+    messages.push_back(msg);
+  }
+  {
+    auto msg = std::make_shared<test_communication::DynamicArrayPrimitives>();
+    msg->bool_values = {true};
```

Comment:
The Windows slave on my desktop is still not working. I wasn't able to get it up and running before I left. The credentials are still the same as stated on the internal wiki if you guys need to fix it while I'm gone. I was going to work on it remotely, but it must have went to sleep or something because I can't seem to connect to it.